base-widget/src/WidgetChannelManager.js

```diff
@@ -120,3 +119,11 @@ export default class WidgetChannelManager {
     }, 1000)
     }
 }
+
+let widgetChannelManager = null;
+
+if(!widgetChannelManager) {
```

Comment:
Let's remove this condition plus the assigning `null` in the previous line. It always comes inside the condition.